benchmarks/cpp/p2p_communication.cpp

```diff
@@ -23,10 +23,10 @@
 namespace nvfuser {
 
 void benchmarkP2PCommunication() {
-  // Test powers of 2 tensor sizes from 2^10 to 2^26
-  std::vector<int> tensor_sizes;
-  for (int power = 10; power <= 26; power++) {
-    tensor_sizes.push_back(1 << power);
+  // Test powers of 2 tensor sizes from 2^10 to 2^32
+  std::vector<long> tensor_sizes;
```

Comment:
```suggestion
  std::vector<int64_t> tensor_sizes;
```
long is platform specific and therefore slightly less reliable